package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "my-webpack",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
```

Comment:
- [ ] Nice job :clap: on writing some scripts, however, I think it would be good if you add the script that will allow your app to start using `npm start` and add instructions in your README file on how to start your App`. Hint :point_down: 

```json
  "scripts": {
    "build": "webpack --config webpack.config.js",
    "start": "webpack serve --open"
  },
```